templates/index.html

```diff
@@ -71,7 +71,11 @@ <h2 class="package--name">{{ package.name }}</h2>
                         <div class="action-bar">
                             <a class="action" href="{{ package.homepage }}">Visit homepage</a>
                             {% if package.last_updated and package.last_update_successful %}
-                            <a class="action" href="{{ package.download_url }}">Download package</a>
+                            <a class="action" href="{{ package.download_url }}">Download package
+                                {% if package.download_count %}
+                                 ({{ package.download_count }} downloads)
+                                {% endif %}
+                            </a>

```

Comment:
its probably best to put this near the Github stars with an icon

Reply:
done